BridgeAppSDK/SBABridgeInfo.swift

```diff
@@ -91,6 +91,32 @@ public protocol SBABridgeInfo: SBASharedAppInfo, SBBBridgeInfoProtocol {
      do not check for a text user.
     */
     var disableTestUserCheck: Bool { get }
+    
+    /**
+     Show a back button in the navigation bar of ORKTaskViewController.
+     */
+    var showBackButtonInNavBar: Bool { get }
```

Comment:
since this is a question, should be `shouldShowBackBarInNavBar`.  Additionally, since it's one or the other (nav bar or footer) then an enum would be more appropriate.  Also, why are these defined at this level at all since there is no code at this level that references them?